binaries/scheduler/main.go

```diff
@@ -17,8 +19,19 @@ func main() {
 	// Create Saga Log
 	// TODO: Replace with Durable SagaLog, currently In Memory Only
 	sagaCoordinator := saga.MakeInMemorySagaCoordinator()
-	sched.NewScheduler(cluster, clusterState, sagaCoordinator)
+	scheduler := sched.NewScheduler(cluster, clusterState, sagaCoordinator)
 
-	// TODO: pull work off of WorkQueue and schedule
+	// TODO: Replace with Durable WorkQueue, currently in Memory Only
+	_, workCh := memory.NewSimpleQueue(1000)
 
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	// Go Routine which takes data from work queue and schedules it
+	go func() {
+		defer wg.Done()
+		sched.GenerateWork(scheduler, workCh)
+	}()
+
+	wg.Wait()
```

Comment:
This seems like it'll have the same result as running this on the same goroutine. Am I missing something?